code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2081,7 +2081,8 @@
 	if(!metal_ref)
 		return
 	var/metal_amount = 0
-
+	if(istype(A,/obj/item/stack/sheet))
```

Comment:
I don't see the need of doing this if you are already making it so you can't sell it.